assets/app/views/browse/build-config.html

```diff
@@ -240,6 +240,18 @@
                                   <dt ng-if="buildConfig.spec.source.contextDir">Source context dir:</dt>
                                   <dd ng-if="buildConfig.spec.source.contextDir">{{buildConfig.spec.source.contextDir}}</dd>
                                 </div>
+                                <div ng-if="buildConfig.spec.source.type == 'None'">
+                                  <dt>Source:</dt>
+                                  <dd>
+                                    Not set
```

Comment:
i would just make this say _none_ here too, that is consistent with what we do elsewhere